.gitignore

```diff
@@ -279,3 +279,4 @@ __pycache__/
 #generated localization files
 Localization/
 **/.DS_Store
+/DFC.ServiceTaxonomy.Theme/wwwroot/Styles/trumbowyg_scoped_govuk_frontend.css.map
```

Comment:
It'll be useful to include the map file.

Reply:
Updated .gitignore file